routes/index.js

```diff
@@ -13,11 +13,12 @@ router.post('/login', user.submit_login)
 router.post('/logout', user.logout)
 router.get('/logout', user.logout)
 //Get account page
-router.get('/account/:user_email', landing.show_account)
+router.get('/account/:user_id', landing.show_account)
+router.post('/account/:user_id', user.edit_profile)
```

Comment:
this is kind of nitpicky, but in terms of REST conventions, i'd probably recommend making this a `put` (or `patch`) request, as that typically indicates wanting to _change_ a record, and `POST` typically indicates _creating_ a record

Reply:
added put request by using the package 'method-override' which allows the put method  from html(jade) to be used.